society/contributors/gregorriegler.md

```diff
@@ -2,9 +2,11 @@
 layout: member
 title: Gregor Riegler
 author: gregorriegler
-role: Technical Coach
+role: Software Development Coach
+affiliation: ebcont
+url: http://gregorriegler.com
```

Comment:
This line and the one above doesn't do anything, the url and affiliation are read from contributors.yml